apps/voting/app/src/components/VoteDetail/DisputableActionStatus.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import {
+  Box,
+  Button,
+  GU,
+  IconLock,
+  Info,
+  textStyle,
+  Timer,
+  useTheme,
+} from '@aragon/ui'
+import { getAgreement } from '../../agreementsMockData'
+import { addressesEqual } from '../../web3-utils'
+
+function DisputableActionStatus({ vote, connectedAccount }) {
+  //TODO: get agreement and vote real data
+  const theme = useTheme()
+  const agreement = getAgreement()
+
+  return (
+    <React.Fragment>
+      <Box heading="Disputable Action Status">
+        <Item>
+          <Label>Status</Label>
+          <div>DisputableStatusTag</div>
+        </Item>
```

Comment:
I think `<Field>` would work nicely for these and keep us visually consistent, it comes with it's own margin too so should slot straight in :)